### PR TITLE
Iow 660 lambda memory usage on dashboard

### PR DESCRIPTION
**Pull request by ssoper-usgs**
Before making a pull request
----------------------------

- [x] Make sure all tests run
- [x] Update the changelog appropriately

Title
-----------
https://internal.cida.usgs.gov/jira/browse/IOW-660

Description
-----------
* Add memory usage widget for each lambda to the dashboard

After making a pull request
---------------------------
- [x] If appropriate, put the link to the PR in the JIRA ticket
- [x] Assign someone to review unless the change is trivial

### CODE REVIEW

File: cloudwatch_monitoring/lambdas.py [hunk 1]
```diff
@@ -234,38 +238,59 @@ def create_lambda_widgets(region, deploy_stage):
...
             elif 'nwis_web' == widget_etl_branch:
                 nwis_web_widgets.append(widget)
                 nwis_web_widgets.append(concurrent_executions_widget)
                 nwis_web_widgets.append(duration_widget)
+                nwis_web_widgets.append(memory_usage_widget)
             else:
                 misc_widgets.append(widget)
                 misc_widgets.append(concurrent_executions_widget)
                 misc_widgets.append(duration_widget)
+                misc_widgets.append(memory_usage_widget)
```
Comment (ayan-usgs) [inline cloudwatch_monitoring/lambdas.py:293]: The list of appends has been increasing. I'm wondering if it's time to consider an "internal function" that does this heavy-lifting.

**Review by ayan-usgs — APPROVED** [review]

File: cloudwatch_monitoring/lambdas.py (hunk 1 above)
Comment (ssoper-usgs) [reply cloudwatch_monitoring/lambdas.py:293]: Agreed, on it.

Comment (ssoper-usgs) [reply cloudwatch_monitoring/lambdas.py:293]: Actually, let me address this in a separate merge.

Comment (ssoper-usgs) [reply cloudwatch_monitoring/lambdas.py]: @ayan-usgs this commit addresses this issue: https://github.com/usgs/aqts-capture-cloudwatch-dashboard/pull/17/commits/bf0dbb7c9d86c3067a3e11221d29fc5bef07d26e#diff-87638053009b1a01dd8e1083c1e380d324408d09057d8ad300e151f8a2b6f0ffR257

**Pull request merged**